mail_as_letter/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
is it possible you included the svg file (which is not necessary if it's just the oca logo unchanged) and did not include the png?